setup.py

```diff
@@ -16,7 +16,7 @@
 
 MACOS_FLAG = ['-mmacosx-version-min=10.7']
 FLAGS_POSIX = [
-    '-fPIC', '-std=gnu++0x', '-Wall', '-Wno-parentheses', '-Werror=switch',
```

Comment:
gnu++0x gives us higher compatibility with older compilers so I would prefer this be rebased out

Reply:
yes, as this commit does not care about backwards compatibility closing it